pkg/kubelet/kubelet.go

```diff
@@ -2055,14 +2069,32 @@ func (kl *Kubelet) isPodResourceUpdateAcceptable(pod *v1.Pod) bool {
 					return false
```

Comment:
do we need a status update here too?

Reply:
since the pod is deleted successfully at this point, there is no need to update status as the pod instance is going away. controller will create a new pod. but i added event logs prior to deletion similar to what is done in scheduler.